src/ios/AppDelegate+FCMPlugin.m

```diff
@@ -298,6 +305,16 @@ - (void)applicationDidEnterBackground:(UIApplication *)application
 }
 // [END disconnect_from_fcm]
 
+- (void)sendDataMessageFailure:(NSNotification *)notification 
+{
+    NSLog(@"Send fail"); 
```

Comment:
Maybe log the content of the notification too and if any error messages for the failure ?

Reply:
I've changed it to also log the notification object. I believe this should also include the error in the `userInfo` property.